angular/src/app/viewer/visualization/network-display/network-display.component.ts

```diff
@@ -88,56 +87,51 @@ export class NetworkDisplayComponent implements AfterViewInit {
 
   ngAfterViewInit(): void {
     console.log('Values at NetDisplayComp on ngAfterViewinit():');
-    console.log('layoutPicker: ', this.layoutPicker);
   }
 
   generateNetwork(): void {
     this.fetchingNetwork = true;
 
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const currentNetworkGeneSet =
+      this.networkGeneSet$.value !== null ? this.networkGeneSet$.value : '';
     const geneSet = this.userData.geneSets$.value.get(
-      this.networkGeneSet$.value!
+      currentNetworkGeneSet //this.networkGeneSet$.value
     );
```

Comment:
Thanks for fixing the linter errors here! I think it might be possible to simplify this logic but I'll think about it and make a PR at some point.